ci/test-conformance-eks.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to be run in build step of a Jenkins job
```

Comment:
it seems to me that it would be fairly straightforward to make the script generic enough so that anyone who has a eksctl installed locally and configured properly would be able to run the script  locally / outside of Jenkins

it would also be more convenient for local testing of this script

Reply:
Sure will abstract some jenkins env vars as params

Reply:
I still see references to jenkins in the script. Maybe the kubeconfig path and output directory could be parameters?